doc/cprover-manual/file2.c

```diff
@@ -7,4 +7,6 @@ int sum()
   sum = 0;
   for(i = 0; i < 10; i++)
     sum += array[i];
+
+  return 0;
```

Comment:
Wouldn't it be more meaningful to return `sum`? (And, as a nit-pick, I don't think this particular change should be in a separate commit, because an earlier commit (wrongly) removed the `return` statement here.)